feature/gribi/otg_tests/backup_nhg_multiple_nh_test/backup_nhg_multiple_nh_test.go

```diff
@@ -333,6 +333,7 @@ func (a *testArgs) testIPv4BackUpSwitch(t *testing.T) {
 	// create flow
 	dstMac := gnmi.Get(t, a.ate.OTG(), gnmi.OTG().Interface(atePort1.Name+".Eth").Ipv4Neighbor(dutPort1.IPv4).LinkLayerAddress().State())
 	BaseFlow := a.createFlow(t, "BaseFlow", dstMac)
+        time.Sleep(5 * time.Second)
```

Comment:
could you please elaborate what is flaky here and how does this sleep help? We do not add sleep in fp tests  and it is better we add checking for preconditions if is required.

Reply:
Script used port counters for verifying traffic flow and port counters can account for control packets too (like arp). This lead to flakiness. Updated the script to use flow counters like in other OTG scripts.